### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by NRKirby**
Hi,

This PR is for [issue #842](https://github.com/serilog/serilog/issues/842). I haven't made any changes to the given templates in the issue. If you want them different in any way pls let me know how...

Nick

### CODE REVIEW

**Comment by nblumhardt** [comment]
Thanks Nick! Sorry to have taken a while to get around to this - someone should be able to review/offer feedback soon. Cheers!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+**What is the new behavior (if this is a feature change)?**
+
+
+
+**Does this PR introduce a breaking change?**
+
+
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit follows our guidelines: https://github.com/serilog/serilog#contribute
```
Comment (merbla) [inline .github/PULL_REQUEST_TEMPLATE.md]: We currently don't have any contribution guidelines :(

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
```
Comment (merbla) [inline .github/PULL_REQUEST_TEMPLATE.md]: Currently we rely on the wiki for our docs.  This is a great check however not sure we could enforce ATM

Comment (nblumhardt) [inline .github/PULL_REQUEST_TEMPLATE.md]: In the sink projects, we could suggest updating the README?

**Comment by nblumhardt** [comment]
Hey @merbla @NRKirby - what do you think about this PR now, with a few months behind us? Are we going to push ahead with it?

I'm keen on the overall idea, but I think we need more discussion about what we want in the templates. Shall we close this and go back to the issue to do that, or iterate here?

**Comment by merbla** [comment]
@nblumhardt yep, I have a few ideas.  I have created some templates however I was trying to come up with Sink templates as well.  

Perhaps we move this discussion to the issue and proceed from there.

**Pull request closed without merging**